MeetupPlatformApi/Persistence/Migrations/20220405110448_AddRefreshToken.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace MeetupPlatformApi.Migrations
+{
+    public partial class AddRefreshToken : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "refresh_tokens",
+                columns: table => new
+                {
+                    id = table.Column<Guid>(type: "uuid", nullable: false, defaultValueSql: "gen_random_uuid()"),
+                    user_id = table.Column<Guid>(type: "uuid", nullable: false, defaultValueSql: "gen_random_uuid()")
```

Comment:
Do we really need automatically generate a value for this field? We can't save refresh token, that is not assigned to a user and it can only be assigned if we fill out this field.